File: Java/src/Act3/Guerrero.java
package Act3;

public class Guerrero extends Personaje {

    public Guerrero(String nombre, int puntosVida, int puntosAtaque) {
        super(nombre, puntosVida, puntosAtaque);
    }

    @Override
    public void atacar(Personaje enemigo) {
        System.out.println(nombre + " ataca con fuerza a " + enemigo.getNombre());
        enemigo.recibirDanio(puntosAtaque);
    }
}

File: KotlinCode/src/Act3/Guerrero.kt
package Act3

class Guerrero(nombre: String?, puntosVida: Int, puntosAtaque: Int) : Personaje(nombre, puntosVida, puntosAtaque) {
    override fun atacar(enemigo: Personaje) {
        println(nombre + " ataca con fuerza a " + enemigo.nombre)
        enemigo.recibirDanio(puntosAtaque)
    }


}

